debian/base/Dockerfile.rpm

```diff
@@ -0,0 +1,94 @@
+#
```

Comment:
Of course, another pr is fine. This will help maintaining things easier in the future.